articles/Intro to C++ - Part 3/index.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/functions-in-cpp/
+title: Functions in C++
+description: Functions in C++, what are they? How do they work? In this article we will be going over one of the most fundamental idea in programming.
+author: Parampreet Singh
+date: 2020-08-30T00:00:00-12:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/functions-in-cpp/hero.jpg
+    alt: functions Function C++ cpp 
+---
+Let us continue from where we left off in the previous article [*Introduction to C++*](/engineering-education/intro-to-c-part1/). We started by going over the importance of C++ as a programming language, and where it is being used today. Most importantly, we tried to go over the basic concepts of the C++ language that any developer would need to get started. If you haven't had a chance to read it already, make sure you do to get a better understanding as we move forward. 
+<!--more-->
+
+## Functions in C++
+
+### **What are Functions?**
+
+Functions have a name and take zero or more parameters. The parameters are how information gets sent to the function. The function only cares about the order of the parameters. It doesn't know or care about the names of the variables in your program. 
+
+The only way to send information to a function is through parameters. And the only way a function can send information back is by returning the value.
```

Comment:
Does not explain what a function does. A better way to start this article would be to place these paragraphs under the paragraph below it.

Reply:
parameters. And the ->  parameters ,and the

Reply:
back is by -> back to the scope above it is by

Reply:
parameters. The parameters are how -> parameters. These parameters are how

Reply:
the parameters. It doesn't -> the parameters, and it doesn't